docs/store-operations/translations/index.mdx

```diff
@@ -0,0 +1,26 @@
+# Translations Admin GraphQL API (Beta)
+
+## Overview
+The Translations Admin GraphQL API is currently available on Catalyst storefronts only. This API enables merchants to translate a single website into multiple languages. All [Translation for Categories (Beta)](/docs/store-operations/translations/categories.mdx) and [Translation for Locations (Beta)](/docs/store-operations/translations/locations.mdx) are translated in the same unified way.
+
+With a single storefront setup, the shopper's experience remains consistent, but the content is displayed in their preferred language. This use case is particularly common in multilingual countries like Canada and for customers selling cross-border, where the same storefront serves shoppers in multiple languages without altering the overall user experience.
+
+## Purpose and Benefits
+The API helps merchants localize their storefronts efficiently, enabling a consistent shopping experience across multiple languages. This allows merchants to cater to a broader audience and expand market reach.
+
+## Authentication and Access
+Access to the Translations Admin GraphQL API requires valid API credentials. GraphQL Admin API endpoints use the X-Auth-Token header to authenticate to BigCommerce servers. To make a call to the Admin GraphQL, use an API client tool such as [Postman](https://www.postman.com/). Ensure authentication tokens are included in requests.
```

Comment:
It would be good to add that the token must include Store Translations permissions.